cmd/posix-errors.go

```diff
@@ -25,8 +25,7 @@ import (
 // Function not implemented error
 func isSysErrNoSys(err error) bool {
 	if pathErr, ok := err.(*os.PathError); ok {
-		switch pathErr.Err {
-		case syscall.ENOSYS:
+		if pathErr.Err == syscall.ENOSYS {
```

Comment:
Further simplification is possible simply do `return pathErr.Err == syscall.ENOSYS`

Reply:
indeed